### PR TITLE
support dynamic max upload size on tus endpoint

### PR DESCRIPTION
**Pull request by kwypchlo**

### CODE REVIEW

**Review by ChrisSchinnerl — APPROVED** [review]

File: docker/nginx/conf.d/client.conf
Comment (ro-tex) [inline docker/nginx/conf.d/client.conf:349]: This hits `accounts` before each upload. Which leads to a DB hit. That's not fast enough to accommodate this use case. We need to cache it somewhere along the line before we can deploy this. I would actually prefer to do it before we even merge this, just to avoid the possibility of accidentally pushing it to prod and melting the DB, thus killing the entire prod.

We can cache it at any point, really, but closest to nginx will be best because of the saved network chatter. If it's hard to add it here, I can add the next best thing in `accounts`.

Comment (ro-tex) [reply docker/nginx/conf.d/client.conf:349]: Added a F/U issue here: https://github.com/SkynetLabs/skynet-webportal/issues/901

Again, I would prefer to not merge this right now in order to avoid the risks outlined above. But there is nothing wrong with the PR, so I'll approve it.

**Review by ro-tex — APPROVED** [review]

**Pull request merged**
